src/main/java/gov/nysenate/openleg/spotchecks/daybreak/DaybreakMessage.java

```diff
@@ -31,8 +31,13 @@ public LocalDateTime getReportDateTime() {
 
     @Override
     public String toString() {
+        String subject = null;
+        try {
+            subject = message.getSubject();
+        }
+        catch (MessagingException ignored) {}
```

Comment:
I'm thinking this check is not necessary. `mesage.getSubject()` is called in the constructor, so if it were to fail the instance would never be created.

Reply:
It's true that this check will always work, but a MessagingException is a checked exception, so it has to be handled regardless.